_inc/client/plans/single-product-search/index.jsx

```diff
@@ -37,20 +69,23 @@ export default function SingleProductSearchCard( props ) {
 				</p>
 				<h4 className="single-product-backup__options-header">
 					{ __(
-						'Your current site record size: %d record',
-						'Your current site record size: %d records',
-						{ args: PLACEHOLDER_RECORD_COUNT, count: PLACEHOLDER_RECORD_COUNT }
+						'Your current site record size: %s record',
+						'Your current site record size: %s records',
+						{ args: recordCount, count: recordCount }
 					) }
 				</h4>
 				<div className="single-product-search__radio-buttons-container">
 					<PlanRadioButton
```

Comment:
@keoshi can we get a different design for what we want to do here. The radio button feels weird unless we are going to have the selection be between monthly and yearly.

For yearly we could then say "two months free" or something and show them side by side.

Reply:
This has since been updated, but I think we'd benefit greatly from some design direction on this part of the interface.

Reply:
Yeah, this is still a long way from an optimal solution, but since it has to be articulated with the Backup team, I'd say let's leave it for now. I will start that conversation with the design team as soon as possible having this mock as a starting point:

![image](https://user-images.githubusercontent.com/390760/77677861-3560e780-6f88-11ea-89d6-2056da32319b.png)